data.json

```diff
@@ -188,6 +188,11 @@
             ]
         }
     },
+    "accessories-ebook-reader": {
+        "linux": {
+            "accessories-ebook-reader"
```

Comment:
This is an object and not a list it should be `"root" : "accessories-ebook-reader"`

Reply:
Oh yeah, all looking too similar to stand out

Reply:
:facepalm: